client/blocks/inline-help/style.scss

```diff
@@ -107,24 +107,8 @@
 
 .inline-help__popover.popover {
 	@include breakpoint( '<660px' ) {
-		top: 0 !important;
```

Comment:
Looks like there are `!important` rules here to override the inline styles. In case folks were wondering why. Hopefully we can build this out in a way to avoid this.

<img width="479" alt="Screen Shot 2020-03-24 at 1 08 59 PM" src="https://user-images.githubusercontent.com/1270189/77472161-bc6b5f80-6dd0-11ea-83f2-14df15f6e34b.png">